tools/benchmark/scripts/run_benchmarks_file.bash

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
This is a lot of bash.  What's the principle underpinning how this will evolve?  If you need to do analysis of the output, bash will quickly hit limits.  

I kind of expected either a go code / go test like structure, or a python / colab style experimentation setup.  Agree some of these things are good for bash, but not all of them are.

Reply:
Bash is mostly kubectl commands to set up and tear down the benchmarks, output analysis is done with a Jupyter notebook.

But agree there is quite a lot of bash, and some of it can/should be done with python or go for better readability and testing. 

I think a key principle to discuss is the [benchmark config API](https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/563/files#diff-5c8d22b3d554a5bc47760ec1d3385360b2f6310038ec2f19de5cedcb83a7e9e2), as well as the benchmark results API (currently implicitly defined by the LPG tool json format). With those defined people can come up with different implementations of automating the benchmark. And bash, or go, or juputer notebook is more of an implementation detail.